scaletest/workspacetraffic/metrics.go

```diff
@@ -0,0 +1,49 @@
+package workspacetraffic
+
+import "github.com/prometheus/client_golang/prometheus"
+
+type Metrics struct {
```

Comment:
Few other metrics that could be useful:
* total execution time for tests
* read, write errors (instead of generic errors)